libfwupdplugin/meson.build

```diff
@@ -8,6 +8,31 @@ fwupdplugin_version_h = configure_file(
   configuration: conf
 )
 
+fwupdplugin_generated_srcs = []
+foreach generated: [
+  'fu-acpi-table',  # fuzzing
+  'fu-self-test',   # fuzzing
+]
+  fwupdplugin_generated_srcs += custom_target('@0@generated'.format(generated),
+    input: '@0@.c'.format(generated),
+    output: '@0@-generated.c'.format(generated),
```

Comment:
rather than parsing comments, how about we specify in a separate XML, JSON, or other directly parsable format?

Reply:
We could do; it was just handy having the definition in the c file where it was being used. Something like `fu-acpi-table.struct` perhaps?